src/integrations/magni.ts

```diff
@@ -0,0 +1,16 @@
+import fetch, { Response } from 'node-fetch';
+import { fetchOptions } from '../http';
+import { FastifyRequest } from 'fastify';
+
+const magniOrigin = process.env.MAGNI_ORIGIN;
+
+export const postFeedback = async (req: FastifyRequest): Promise<Response> =>
```

Comment:
It's better to be more type-safe than this. I suggest that this function will accept the exact parameters expected by magni instead of relying on the req object.